nconf/configuration.go

```diff
@@ -1,15 +1,61 @@
 package nconf
 
 import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
 	"os"
+	"path/filepath"
 	"strings"
 
+	"github.com/netlify/netlify-commons/featureflag"
+	"github.com/netlify/netlify-commons/metriks"
+	"github.com/netlify/netlify-commons/tracing"
```

Comment:
if you want to split these off, they usually go below other packages. i wish gofmt enforced this

Reply:
yes I wish that it would do that automatically. I just lumped them together for the time being so fmt can order them nicely